ci/packet-arm/packet-arm-cluster.lokocfg.envsubst

```diff
@@ -27,6 +27,7 @@ cluster "packet" {
   node_private_cidr = "10.0.0.0/8"
 
   worker_pool "pool-1" {
+    ssh_pubkeys       = ["$PUB_KEY"]
```

Comment:
I wonder if this is against the idea from #358, where we want to remove the duplication of SSH keys.

Reply:
I think we should allow setting SSH keys per worker pool, but by default, they should just inherit "default" keys somehow.

Reply:
Added a method that inherits the ssh keys, tags and clc snippets from controller if not provided by the user.